src/pool/rpc.h

```diff
@@ -64,6 +64,9 @@
 	X(POOL_EXCLUDE,							\
 		0, &CQF_pool_exclude,					\
 		ds_pool_update_handler, NULL),				\
+	X(POOL_EXTEND,							\
+		0, &CQF_pool_extend,					\
+		ds_pool_update_handler, NULL),				\
```

Comment:
There's a follow up PR for the actual implementation of the pool extend process getting reviewed right now. I broke the DMG stuff off into it's own patch so it would be easier to review/land.

I'm working on both so it shouldn't fall through the cracks if that's what you meant.